src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const totalCost = days * dailyRate;
+  const smallDiscount = 20;
+  const discount = 50;
+  let cost = 0;
+
+  if (days < 3) {
+    return totalCost;
+  }
+
+  if (days < 7) {
+    cost = totalCost - smallDiscount;
+
+    return cost;
```

Comment:
Similarly, use a constant to represent the minimum days required for the larger discount (e.g., `const MIN_DAYS_FOR_LARGE_DISCOUNT = 7`). This will align with the checklist item of not using literals in expressions.